spec/unit/log_spec.rb

```diff
@@ -23,21 +23,23 @@
 describe Chef::Log do
 end
 
-describe Chef::Log::Syslog do
-  let(:logger) { Chef::Log::Syslog.new }
+unless Chef::Platform.windows?
```

Comment:
In this case you can't use the syslog-logger gem on windows at all which makes it difficult to write a unit test that works on windows (not without getting crazy and stubbing require to return true and other horrible things).